src/core.js

```diff
@@ -13,6 +13,10 @@ import { sourceFromStacktrace } from "./core/stacktrace";
 
 const QUnit = {};
 
+// take a predefined QUnit.config and extend the defaults
+var globalConfig = window && window.QUnit && window.QUnit.config;
+extend( config, globalConfig );
```

Comment:
This should go in `src/core/config.js` instead. Easier to reason about what is happening that way.